sc.js

```diff
@@ -0,0 +1,81 @@
+var sauceConnectLauncher = require('sauce-connect-launcher')
```

Comment:
I would say the name of the file is too short and thus not descriptive enough for the root directory.